gratipay/utils/team_history.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+from decimal import Decimal as D
+
+from aspen import Response
+
+
+def get_end_of_year_totals(db, team, year):
+
+    received = db.one("""
```

Comment:
@whit537 @kaguillera a general question, is it worthwhile/feasible to use some database-abstraction api instead of raw sql statements?